module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
+
+Now lets create a directory to store your AWS CDK Application. 
+
+```bash
+mkdir CDK
+cd CDK
 ```
-aws s3api list-buckets --query 'Buckets[?starts_with(Name, `mythical-mysfits-frontend`) == `true`]'
+
+One of the biggest benefits from AWS CDK is the principal of reusability - Being able to write, reuse and share components throughout your application and team.  These components are referred to as Constructs within AWS CDK.  To this end, the code we will write in Module 1 will be reused throughout all remaining modules.
+
+#### Install AWS CDK
+
+Install the CDK using the following command.
+
+```bash
+npm install -g aws-cdk
 ```
+
+Run the following command to see the version number of the CDK.
+
+```bash
+cdk --version
 ```
-Get-S3Bucket | Where-Object { $_.BucketName -Like "mythical-mysfits-frontend*" }
+
+### Initialise CDK Application
+
+Initialize an app, where LANGUAGE is one of the supported programming languages: csharp (C#), java (Java), python (Python), or typescript (TypeScript) and TEMPLATE is an optional template that creates an app with different resources than the default app that cdk init creates for the language.
+
+```bash
+cdk init app --language LANGUAGE
 ```
-#### (Optional) Use AWS Amplify to Configure Website Hosting
-Instead of using the deploy scripts in the `deploy-frontend-scripts` folder, we can use a toolchain called [AWS Amplify](https://aws-amplify.github.io/). AWS Amplify is a library and CLI tool built to help quickly create, deploy, and use backend services with client applications in JavaScript, iOS, and Android.
 
-We'll use AWS Amplify later in the tutorial to add authentication, user sign-in, and API access, but the AWS Amplify CLI also includes a feature for `hosting`.
+For the purposes of this workshop we will use TypeScript as our language.  So execute the following command
+
+```bash
+cdk init app --language=typescript
+```
 
-**It's important to note:** Only choose one method of deployment – either use the deployment scripts or AWS Amplify.
+This command will initialise a new CDK application.
 
-Follow these minimum steps for working with the AWS Amplify CLI, or view the full [Getting Started guide](https://aws-amplify.github.io/amplify-js/media/quick_start?platform=purejs).
+### Implementation
+
+Now, let's implement the code to host our web application.  Open the `CDK` folder in VS Code.
+
+The standard structure of an AWS CDK application consists of a `bin` folder and a `lib` folder.
+
+* The `bin` folder is where we will define the entry point for the CDK application.
+* The `lib` folder is where we will define all our workshop infrastructure components.
+
+before we start implementing our code, let's get typescript to watch for file changes and compile the javascript whenever a change is detected by running the following command from the terminal.
+
+```bash
+npm run watch
```

Comment:
That's actually not needed anymore since `cdk.json` now uses `ts-node` which compiles the code just in time

Reply:
This was more to provide realtime feedback whilst the developer is coding.  Still remove?